spec/integration/date_dim_switch_spec.rb

```diff
@@ -135,10 +139,10 @@ def suggest_mapping(label_a, label_b, project)
     expect(@report.definition.attributes.map(&:identifier)).to eq ["created_on_2.quarter"]
 
     ids = GoodData::SmallGoodZilla.get_uris(@report.definition.filters.first).map { |x| x.split('/')[-2..-1].join('/') }
-    expect(ids).to eq ["obj/281", "281/elements?id=2015", "281/elements?id=2016"]
+    expect(ids).to eq ["obj/#{year_obj_id}", "#{year_obj_id}/elements?id=2015", "#{year_obj_id}/elements?id=2016"]
 
     ids = GoodData::SmallGoodZilla.get_uris(@variable.values.first.expression).map { |v| v.split('/')[-2..-1].join('/') }
-    expect(ids).to eq ["obj/281", "281/elements?id=2015", "281/elements?id=2016"]
+    expect(ids).to eq ["obj/#{year_obj_id}", "#{year_obj_id}/elements?id=2015", "#{year_obj_id}/elements?id=2016"]
```

Comment:
Identical code found in date_dim_switch_spec.rb:142, date_dim_switch_spec.rb:145 (mass = 64)